code_samples/back_office/udw/assets/js/image-tab/components/images.list.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useContext, useEffect } from 'react';
 import Image from './image';
 import { getImages } from '../services/images.service';
 
-import { RestInfoContext } from '../../../../vendor/ezsystems/ezplatform-admin-ui/src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module';
+import { RestInfoContext } from '../../../../vendor/ibexa/admin-ui/src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module';
```

Comment:
Now you can use just:
```suggestion
import { RestInfoContext } from '@ibexa-admin-ui/src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module';
```